lib/git-hooks.js

```diff
@@ -160,11 +166,22 @@ function isExecutable(stats) {
  */
 function spawnHook(hookName, args) {
     var stats = fs.statSync(hookName);
-    var isHookExecutable = stats && stats.isFile() && isExecutable(stats);
-    if (!isHookExecutable) {
+    var command = hookName;
+    var opts = args;
+    var hook;
+
+    if (isWin32) {
+        hook = fs.readFileSync(hookName).toString();
+        if (!require('shebang-regex').test(hook)) {
```

Comment:
I don't merge the pr with extra modules.

If you really want to help, let's list all current problems with windows version and find out the best solution for each of them. Discussion first, then code.

We can start with your description: https://github.com/tarmolov/git-hooks-js/issues/10#issuecomment-172345975